MyHashMap.java

```diff
@@ -0,0 +1,45 @@
+
+// Time Complexity : O(n)
```

Comment:
Since you are accessing the elements without traversing all the elements, the time complexity would be O(1)

Also, try the solution taught in class